src/main/groovy/org/labkey/gradle/task/CreateModule.groovy

```diff
@@ -0,0 +1,169 @@
+package org.labkey.gradle.task
+
+import org.gradle.api.DefaultTask
+import org.gradle.api.file.CopySpec
+import org.gradle.api.tasks.TaskAction
+
+import java.util.regex.Matcher
+import java.util.regex.Pattern
+
+/**
+ * Creates a new module based on the contents of trunk/server/moduleTemplate. Sources properties from commandline or
+ * user input based on the prompt flag.
+ */
+class CreateModule extends DefaultTask
+{
+    boolean prompt = true
+
+    @TaskAction
+    void createModule() {
+        String moduleName = null
+        String moduleDestination = null
+        boolean hasManagedSchema = true
+        boolean createTestFiles = true
+        boolean createApiFiles = true
```

Comment:
Set these values from the properties, if they exist. Then check for all the required properties to determine if the prompt is needed instead of passing that in as a parameter.  

The default values for createTestFiles and createApiFiles should be false to be consistent with the ant task.